app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,31 @@
  *= require foundation_and_overrides
 
  */
+
+ .profile-photo img {
+  width: 3em;
+  max-height: 3em;
+  float: right;
+  margin-top: 2px;
+  margin-right: 0px;
+  background: inherit;
+}
+
+.profile-photo li {
+  background: #000000;
+}
+
+.user-pic img {
+ width: 3em;
+ max-height: 3em;
+ margin-top: 2px;
+ margin-right: 0px;
+ // background: inherit;
+}
+
+.user-name {
+  float: left;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.